src/hub/Hub.sol

```diff
@@ -309,33 +309,55 @@ contract Hub is Multicall, Auth, Recoverable, IHub, IHubGatewayHandler {
         PoolId poolId,
         ShareClassId scId,
         AssetId assetId,
-        IERC7726 valuation,
-        bool isLiability,
-        uint24 prefix
+        IERC7726 valuation,AccountId assetAccount, AccountId equityAccount, AccountId lossAccount, AccountId gainAccount
```

Comment:
NIT. To fix the format of this file, you can:

- comment `transient` keywords
- Uncomment the file in `foundry.yml`
- save/compile again the fail with the format
- comment the file in `foundry.yml`
- comment `transient` keywords